src/common/routes.js

```diff
@@ -109,32 +109,43 @@ export const conferenceEventsHandler = async (req, res) => {
       .conferences(ConferenceSid)
       .participants.list();
 
-    const isParticipantLeave = StatusCallbackEvent === 'participant-leave';
-
     // Send the conference sid and the other participants' callSids to any client leg.
     // The sids will be used to update the participant resource such as putting it on hold.
-    participants.forEach((currentParticipant) => {
+    participants.forEach(async (currentParticipant) => {
       if (currentParticipant.label.split('-')[0] === 'client') {
-        participants
-          .filter((p) => p.callSid !== currentParticipant.callSid)
-          .forEach(async (otherParticipant) => {
-            await client
-              .calls(currentParticipant.callSid)
-              .userDefinedMessages.create({
-                content: JSON.stringify({
-                  conferenceSid: ConferenceSid,
-                  callSid: isParticipantLeave
-                    ? CallSid
-                    : otherParticipant.callSid,
-                  label: isParticipantLeave
-                    ? ParticipantLabel
-                    : otherParticipant.label,
-                  muted: isParticipantLeave ? Muted : otherParticipant.muted,
-                  hold: isParticipantLeave ? Hold : otherParticipant.hold,
-                  remove: isParticipantLeave,
-                }),
-              });
-          });
+        if (StatusCallbackEvent === 'participant-leave') {
```

Comment:
Add a comment above the `if` and `else` statement and describe what these blocks are doing. See lines 112-113 above for an example.